apple/internal/aspects/swift_dynamic_framework_aspect.bzl

```diff
@@ -115,6 +114,7 @@ single swift_library dependency.\
                     continue
                 swiftmodule = module.swift.swiftmodule
                 swiftdoc = module.swift.swiftdoc
+                module_name = module.name
```

Comment:
What this change intentional?

Reply:
Yes, needed to get the head deps job passing

Reply:
:+1: Not in master.